lib/puppet_x/puppetlabs/registry.rb

```diff
@@ -6,42 +6,38 @@ module Registry
   # For 64-bit OS, use 32-bit view. Ignored on 32-bit OS
   KEY_WOW64_32KEY = 0x200 unless defined? KEY_WOW64_32KEY
 
-  # Helper methods used in the registry providers
-  # Guarded to only load on Windows due to Win32 constants
-  if Puppet.features.microsoft_windows?
```

Comment:
You mentioned a concern about specs at https://github.com/puppetlabs/puppetlabs-registry/pull/146#discussion_r162552447 @glennsarti ... but I don't see a problem in Travis.

Reply:
Hrmm...I did break things but perhaps I was overzealous in my fixing.  If it passes Travis, ship it.